src/Components/EmbeddedLogsExploration/EmbeddedLinkScene.tsx

```diff
@@ -26,7 +26,7 @@ export class EmbeddedLinkScene extends SceneObjectBase {
 
     // Rerender this scene whenever any dependent variables are updated
```

Comment:
Outdated comment.

Reply:
Changed in 07300e948fbe1d8275700fa58ad7c8650cbeee0a